viafabric-mc114/src/main/java/com/viaversion/fabric/mc114/providers/VRPlayerLookTargetProvider.java

```diff
@@ -0,0 +1,22 @@
+package com.viaversion.fabric.mc114.providers;
+
+import com.viaversion.viaversion.api.connection.UserConnection;
+import com.viaversion.viaversion.api.minecraft.Position;
+import com.viaversion.viaversion.protocols.protocol1_13to1_12_2.providers.PlayerLookTargetProvider;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.util.hit.BlockHitResult;
+import net.minecraft.util.hit.HitResult;
+import net.minecraft.util.math.BlockPos;
+
+public class VRPlayerLookTargetProvider extends PlayerLookTargetProvider {
+
+    @Override
+    public Position getPlayerLookTarget(UserConnection info) {
+        final HitResult crosshairTarget = MinecraftClient.getInstance().crosshairTarget;
```

Comment:
should check if info == client

Reply:
oh i see, done

Reply:
Should I then delete the checks in the VFLoader classes?